recipes/pythran/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "pythran" %}
+{% set version = "0.8.5" %}
+{% set sha256 = "92f67171d03ba3dbed9da639c35aad2fe0982c183f0c4459dcfddd621f69795e" %}
+{% set variant = "openblas" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  skip: true  # [win]
+  features:
+    - blas_{{ variant }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - gmp
+    - numpy 1.13.*
+    - pytest-runner
+  run:
+    - python
+    - gmp
+    - ply
+    - networkx
+    - {{ pin_compatible('numpy') }}
+    - six
+    - gast
+
+test:
+  files:
+    - dprod.py
+  requires:
+    - {{ compiler('cxx') }}
+  commands:
+    - pythran dprod.py
+    - python -c 'import dprod'
+
+about:
+  home: http://github.com/serge-sans-paille/pythran
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'a claimless python to c++ converter'
+
+  description: |
+    Pythran is an ahead of time compiler for a subset of the Python language,
+    with a focus on scientific computing. It takes a Python module annotated
+    with a few interface description and turns it into a native Python module
+    with the same interface, but (hopefully) faster.
+  doc_url: https://pythran.readthedocs.io/
+  dev_url: https://github.com/serge-sans-paille/pythran
+
+extra:
+  recipe-maintainers:
+    - saraedum
+    - serge-sans-paille
```

Comment:
@serge-sans-paille, do you want to be a maintainer for this recipe?

Reply:
Along with @saraedum , yes!